src/utils.cpp

```diff
@@ -1628,6 +1629,179 @@ int get_skill(struct char_data *ch, int skill, int &target)
     }
   }
 
+  // Reflex Recorder, adds +1 die to Physical Skill tests for its associated skill. -Vile
+  // Utilizes Value 6 to reference its associated skill.
+  if (recorder)
```

Comment:
Not having brackets to mark your code is going to cause a lot of hidden bugs. You should always put {} around your statements to avoid logical flow issues that are hard to diagnose.